### ISSUE TITLE
CustomTabTest fails because webview element intermittently becomes NAF

### ISSUE THREAD

**Issue by npark-mozilla** [mozilla-mobile/focus-android#1200]
Since #960, CustomTabTest fails to detect the element inside webview when it runs after other tests.  hierarchydump shows that the webview element became NAF (Not Accessibility Friendly) . On BB this only fails on API 23, but locally it fails on API 25 and other devices as well.

It appears that leftover from previous tests are affecting this test.  When it runs on its own, it always passes.
Curiously in the subsequent test (which is a non-custom view) the webview elements are properly exposed.  I've read that android webviews sometimes become NAF when it's scrolling or hasn't finished drawing, but this time it seems that's not the case.
